examples/dataproc_provisioner_example/README.md

```diff
@@ -0,0 +1,12 @@
+# Dataproc Compute Profile with Yarn Container Logs in StackDriver
+
+This example illustrates how to use the `dataproc_provisioner` module.
+
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
You can run `sudo make generate_docs` to generate the docs here. Will need to install docker first.